tools/make_coupler_mosaic/make_coupler_mosaic.c

```diff
@@ -838,15 +854,16 @@ int main (int argc, char *argv[])
 	if(!is_uniform && mpp_pe()==mpp_root_pe()) printf("\nNOTE from make_coupler_mosaic: ocean grid latitude is not uniform along j = 1\n");
 
 	/* calculate the minimum of latitude of atmosphere grid */
-	min_atm_lat = 9999; /* dummy large value */
-	for(na=0; na<ntile_atm; na++) {
-	  min_lat = minval_double((nxa[na]+1)*(nya[na]+1), yatm[na]);
-	  if(min_atm_lat > min_lat) min_atm_lat = min_lat;
-	}
+	//min_atm_lat = 9999; /* dummy large value */
+	//for(na=0; na<ntile_atm; na++) {
+	//  min_lat = minval_double((nxa[na]+1)*(nya[na]+1), yatm[na]);
+	//  if(min_atm_lat > min_lat) min_atm_lat = min_lat;
```

Comment:
If you agree it's not needed I'll remove it.

Reply:
I dont think its needed, but we have these various version forever in  github.
If it could be an option in the foreseeable future, I think then a comment is appropriate